.github/ISSUE_TEMPLATE/Documentation_Issue.md

```diff
@@ -2,7 +2,7 @@
 name: Documentation Issue 📚
 about: File issues regarding documentation within the [PowerShell-Docs](https://github.com/MicrosoftDocs/PowerShell-Docs) repository
 title: "Documentation Issue"
-labels: CL-Docs
+labels: Issue-Question,Area-Documentation
```

Comment:
Not sure, but I think this might need to be an array or just 1 label.

Reply:
if I remember the syntax right this needs to look more like this:
```suggestion
labels: [Issue-Question, Area-Documentation]
```

Reply:
Working sample https://raw.githubusercontent.com/lee-dohm/test-repo/master/.github/ISSUE_TEMPLATE/Bug_report.md